helm/tests/ensure.py

```diff
@@ -346,3 +347,29 @@ def azurecluster(kubernetes_cluster):
 
     kubernetes_cluster.kubectl(f"delete azurecluster {cluster_name}", output=None)
     LOGGER.info(f"AzureCluster {cluster_name} deleted")
+
+
+# Silence fixtures
+
+@pytest.fixture
+def silence(kubernetes_cluster):
+    c = dedent(f"""
+        apiVersion: monitoring.giantswarm.io/v1alpha1
+        kind: Silence
+        metadata:
+          name: {silence_name}
+          namespace: monitoring
```

Comment:
I don't think this namespace exists in the KIND cluster - using `default` should be fine